flink-state-backends/flink-statebackend-rocksdb/src/main/java/org/apache/flink/contrib/streaming/state/AbstractRocksDBState.java

```diff
@@ -61,8 +64,7 @@
 	/** The column family of this particular instance of state. */
 	protected ColumnFamilyHandle columnFamily;
 
-	/** State descriptor from which to create this state instance. */
-	protected final SD stateDesc;
```

Comment:
Same as for the heap state also applies here, you can remove `SD` from the generic types of this class and the subclasses.

Reply:
Fixed, thanks for catching this.